assets/css/_geolocation.scss

```diff
@@ -33,7 +26,26 @@
   }
 
   .leaflet-container {
-    border: 4.5px solid #000;
     height: 500px;
   }
+
+  .geolocation-button {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 36px;
+    height: 36px;
+    border: 0;
+    font-size: 14px;
+    z-index: 1002;
+    background: transparent;
+    color: #402020;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
```

Comment:
For `user-select`, we use autoprefixer to automatically add vendor prefixes to this property. This is where we're doing it: https://github.com/streetmix/streetmix/blob/master/assets/css/_mixins.scss#L9

Learn more here: https://github.com/postcss/autoprefixer